blog/terraform-post.md

```diff
@@ -0,0 +1,132 @@
+---
+title: AEP Terraform Provider
+date: 2025-03-31
+authors:
+  - name: Alex Stephen
+  - name: Marsh Gardiner
+---
+
+# AEP Terraform Provider
+
+Author: Alex Stephen, Marsh Gardiner
+
+A core thesis of the AEP project has been that API design constraints can
+facilitate the creation (and maintenance\!) of API client tools. In many
+companies, teams are staffed to build out CLI tools, Terraform providers, UIs,
+LLM agents, and various other tools that act as thin layers on top of a
+company's APIs. The AEP project aims to ensure that these tools can be easily
+generated on top of our API design, ensuring that companies can move faster
+with less overall effort. We’re excited to launch our first version of a
+[fully auto-generated Terraform provider](https://github.com/aep-dev/terraform-provider-aep)
```

Comment:
not related to this content specifically, but I'd at least put a 0.1 version tag on it so consumers and understand it's reached some level of stability (even if that's still not too stable)